terra-jupyter-aou/Dockerfile

```diff
@@ -58,11 +58,11 @@ RUN find $JUPYTER_HOME/scripts -name '*.sh' -type f | xargs chmod +x \
     && update-alternatives --install /etc/hadoop/conf hadoop-conf /etc/hadoop/conf.empty 100 \
     && update-alternatives --install /etc/hive/conf hive-conf /etc/hive/conf.dist 100 \
     && apt-get update \
-    && apt install -yq --no-install-recommends openjdk-8-jdk \
-        g++ \
-        liblz4-dev \
-    # specify Java 8
-    && update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java \
+    # Note that starting on 2.1.x, dataproc VM also provides java 11, which is incompatible with 8. No need to install it in the container
+    # && apt install -yq --no-install-recommends openjdk-8-jdk \
```

Comment:
does gatk work with java 11 now?

I vaguely remember gatk was the reason why we used java8

Reply:
Hail is built with java8, which was the original reason why we put this here, but it does work with Java 11 as well, as long as we don't mix and match java versions (I tested this with the terra hail image and it works fine)

Reply:
gatk also needs java...

we should make sure [this version](https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-gatk/Dockerfile#L37) works with java 11

Reply:
Do you have a way to test the aou image on a BEE? So far we tested GATK on terra and it is working, but we don't have a good way to test the AOU image